src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -24,14 +26,50 @@
  *
  * @author Kuanhung Chen <ericj.tw@gmail.com>
  */
-final class MethodArgumentSpaceFixer extends AbstractFixer
+final class MethodArgumentSpaceFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
     /**
+     * @var array<string, bool>
+     */
+    private static $defaultConfiguration = array('keepMultipleSpacesAfterComma' => false);
+
+    /**
+     * Preserve existing multiple spaces after comma.
+     *
+     * @var bool
+     */
+    private $keepMultipleSpacesAfterComma;
+
+    /**
+     * @param null|array $configuration
+     *
+     * @throws InvalidFixerConfigurationException
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            $configuration = self::$defaultConfiguration;
+        }
+
+        if (!array_key_exists('keepMultipleSpacesAfterComma', $configuration)) {
+            throw new InvalidFixerConfigurationException($this->getName(), 'Missing "keepMultipleSpacesAfterComma" configuration.');
```

Comment:
hmm I'm puzzled as well, in other rules we don't take the default config and override with what is set/passed. Either someone runs with default, or has to provide config for all.
Please provide a (pseudo) sample of how you would want this resolved (a link to another fixer is fine as well ;))